docs/guides/setup.md

```diff
@@ -197,7 +197,7 @@ $ sudo systemctl start signctrl
 and then, start your validator via
 
 ```shell
-$ sudo systemctl simd start
```

Comment:
Right, one has to copy the `validator_laddr` field from the `[init]` section in SignCTRL's config.toml into the `laddr` field from the `rpc` section in simd's config.toml.

Reply:
Okay